Libplanet.Stun/Stun/TurnClient.cs

```diff
@@ -218,6 +219,30 @@ public void Dispose()
             }
         }
 
+        public async Task BindingProxies(
+            CancellationToken cancellationToken,
+            int? listenPort)
+        {
+            while (!cancellationToken.IsCancellationRequested)
+            {
+#pragma warning disable IDE0067 // We'll dispose of `stream` in proxy task.
+                NetworkStream stream = await AcceptRelayedStreamAsync();
+#pragma warning restore IDE0067
+
+                // TODO We should expose the interface so that library users
+                // can limit / manage the task.
+                Func<Task> f = async () =>
```

Comment:
If you're going to separate this, how about using a local function or method with a proper name?